dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -47,6 +47,17 @@ HRESULT CFSDropTarget::_CopyItems(IShellFolder * pSFFrom, UINT cidl,
     /* Build a double null terminated list of C strings from source paths */
     for (UINT i = 0; i < cidl; i++)
     {
+        SFGAOF rfg = SFGAO_FILESYSANCESTOR | SFGAO_FILESYSTEM;
+        ret = pSFFrom->GetAttributesOf(1, apidl[i], &rfg);
```

Comment:
`no known conversion for argument 2 from 'LPCITEMIDLIST' {aka 'const _ITEMIDLIST*'} to 'PCUITEMID_CHILD_ARRAY' {aka 'const _ITEMIDLIST* const*'}`

Reply:
Fixed.